src/overlays/actors/ovl_En_Hgo/z_en_hgo.c

```diff
@@ -107,59 +118,59 @@ void EnHgo_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80BD03EC(EnHgo* this) {
+void EnHgo_SetupIdle(EnHgo* this) {
     this->actor.flags &= ~ACTOR_FLAG_1;
-    this->actionFunc = func_80BD0410;
+    this->actionFunc = EnHgo_Idle;
 }
 
-void func_80BD0410(EnHgo* this, PlayState* play) {
+void EnHgo_Idle(EnHgo* this, PlayState* play) {
 }
 
-void func_80BD0420(EnHgo* this) {
-    this->actionFunc = func_80BD0434;
+void EnHgo_SetupInitCollision(EnHgo* this) {
+    this->actionFunc = EnHgo_InitCollision;
 }
 
-void func_80BD0434(EnHgo* this, PlayState* play) {
+void EnHgo_InitCollision(EnHgo* this, PlayState* play) {
```

Comment:
```suggestion
void EnHgo_UpdateCollision(EnHgo* this, PlayState* play) {
```
Gets called in Update so seems more appropriate that it is updating the collision.